### PR TITLE
Set vitest coverage

### PR DESCRIPTION
**Pull request by Vithop**
Set coverage to not fail builds but we should strive to reach 80 or 90 percent coverage

### CODE REVIEW

File: .github/workflows/svelte.yml [hunk 1]
```diff
@@ -10,47 +10,19 @@ defaults:
...
-    steps:
-      - uses: DerYeger/yarn-setup-action@master
-        with:
-          node-version: 16
   build:
-    name: 'Build'
-    runs-on: ubuntu-latest
-    needs: prepare
-    steps:
-      - uses: DerYeger/yarn-setup-action@master
```
Comment (Vithop) [inline .github/workflows/svelte.yml:17]: braaa my beautiful concurrent build process. 😭

**Review by Vithop — COMMENTED** [review]
I guess it's better than my broken workflow

**Comment by Vithop** [comment]
@Khufu-I u need to approve this PR since it's still my PR despite it being ur code

File: .github/workflows/svelte.yml (hunk 1 above)
Comment (Khufu-I) [reply .github/workflows/svelte.yml:17]: The individual steps are so fast that actually setting up the environment for each concurrent build step takes much longer.

**Comment by Khufu-I** [comment]
@Vithop reformat the code and re-enable the lint step in a separate pull request.

**Review by Khufu-I — APPROVED** [review]

**Pull request merged**